src/sentry/data_export/endpoints/data_export.py

```diff
@@ -56,7 +56,11 @@ def validate(self, data):
                 query_info["project"] = [project.id for project in projects]
 
             # make sure to fix the export start/end times to ensure consistent results
-            start, end = get_date_range_from_params(query_info)
+            try:
+                start, end = get_date_range_from_params(query_info)
+            except InvalidParams as e:
+                raise serializers.ValidationError(str(e))
```

Comment:
Just so we can track how often this happens:
```suggestion
                sentry_sdk.set_tag("query.error_reason", "Invalid date params")
                raise serializers.ValidationError(str(e))
```